src/commands/docker-build.yml

```
@@ -26,6 +26,12 @@ parameters:
   use-docker-layer-caching:
     type: boolean
     default: true
+  extra-build-args:
+    default: '--build-arg ssh_prv_key="$(cat ~/.ssh/id_rsa)"'
+    type: string
+  additional-tags:
+    type: string
+    default: ""
```

Lets add a description that tags can be space seperated.
```suggestion
    default: ""
    description: Space separated names for additional images tags.
```